Product/Production/Adapters/General/CONNECTDirectConfig/pom.xml

```diff
@@ -79,6 +79,7 @@
         <dependency>
             <groupId>org.apache.santuario</groupId>
             <artifactId>xmlsec</artifactId>
+            <version>2.0.7</version>
```

Comment:
Why is the version controlled here, instead of in dependencyManagement in the Product POM?  Do CONNECT and DirectConfig need different versions?